h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -219,7 +276,12 @@ public static double l2norm(double [] x, boolean skipLast){
 
   public static double[] add(double[] a, double[] b) {
     if( a==null ) return b;
-    for(int i = 0; i < a.length; i++ ) a[i] += b[i];
```

Comment:
this changes the contract of the method, since the original only modified a now it can modify b as well and deal with different length arrays

Reply:
@valenad1 Is this resolved?

Reply:
I have implemented new addSave method

Reply:
Cool, thanks!